servers/rendering/renderer_rd/shaders/sky.glsl

```diff
@@ -202,20 +202,10 @@ void main() {
 #endif
 #endif
 
-// unused, just here to make our compiler happy, make sure we don't execute any light code the user adds in..
-#ifndef REALLYINCLUDETHIS
 	{
-		/* clang-format off */
-
-LIGHT_SHADER_CODE
-
-		/* clang-format on */
-	}
-#endif
```

Comment:
Nice to see that we can delete this now.

Reply:
oh.. yeah :sweat: